test/fika/parser_test.exs

```diff
@@ -401,6 +401,50 @@ defmodule Fika.ParserTest do
     end
   end
 
+  describe "tuple" do
+    test "parses tuple with one element" do
+      str = """
+      {1}
+      """
+
+      {:ok, result, _rest, _context, _line, _byte_offset} = Parser.expression(str)
+      assert result == [{:tuple, {1, 0, 3}, [{:integer, {1, 0, 2}, 1}]}]
+    end
+
+    test "parses tuple with multiple elements" do
```

Comment:
Should there be a test that shows empty tuples aren't allowed?

Reply:
empty tuple is `{}`, but that's already inferred as empty record `{}` which by itself is not allowed 😄

Reply:
Nice!